### PR TITLE
Run subsystem tests for onprem

### PR DESCRIPTION
**Pull request by rwsu**
Adds back Jenkinsfile changes from PR#46 that were removed in PR#62.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -40,6 +41,19 @@ pipeline {
...
         }
 
+		stage('test subsystem with podman') {
+				environment {
+					SERVICE = 'ocpmetal/assisted-service:test-onprem'
+				}
+			steps {
+				sh '''export PATH=$PATH:/usr/local/go/bin; make build-onprem'''
+				sh 'make deploy-onprem'
+				sleep 30
```
Comment (avishayt) [inline Jenkinsfile:51]: What's this for?  Please add a comment, or preferably have the code wait on something rather than sleep for a fixed time.

**Comment by ronniel1** [comment]
@rwsu , @avishayt , @filanov , @romfreiman , @mhrivnak I'm not sure that we want to run the disconnected in subsystem of every PR.
As it is now, the subsystem takes 20-30 minutes. Adding this will more than double it.

Maybe it should be run on a periodic basis, from master and not on every PR?

**Comment by ronniel1** [comment]
@rwsu Notice that in the pr-merge jenkins run, for some reason the podmen stage did not run

**Comment by ronniel1** [comment]
https://issues.redhat.com/browse/MGMT-1741 will allow to run the onprem subsystem in parallel to the regular subsystem.
Once the is in, we can make this part of the PR

**Comment by rwsu** [comment]
Closed, replaced by https://github.com/openshift/assisted-service/pull/692

**Pull request closed without merging**